controllers/home/index.js

```diff
@@ -20,26 +16,44 @@ async function injectHomepageContent(req, res, next) {
         });
         next();
     } catch (error) {
+        res.locals.promotedUpdates = [];
         Raven.captureException(error);
         next();
     }
 }
 
-router.get('/', injectHomepageContent, (req, res) => {
-    const fallbackSuperheroImage = {
-        default: {
+async function injectHeroImages(req, res, next) {
+    try {
+        const response = await contentApi.getHomepage({ locale: req.i18n.getLocale() });
+        res.locals.heroImages = response.heroImages;
+        next();
+    } catch (error) {
+        res.locals.heroImages = {
             small: '/assets/images/hero/superhero-fallback-small.jpg',
             medium: '/assets/images/hero/superhero-fallback-medium.jpg',
             large: '/assets/images/hero/superhero-fallback-large.jpg',
             default: '/assets/images/hero/superhero-fallback-medium.jpg',
             caption: 'Stepping Stones Programme, Grant £405,270'
-        }
-    };
+        };
+        Raven.captureException(error);
+        next();
+    }
+}
 
-    res.render(path.resolve(__dirname, './views/home'), {
-        news: get(res.locals, 'newsArticles', []),
-        heroImage: get(res.locals, 'heroImages', fallbackSuperheroImage)
-    });
+router.get('/', injectPromotedUpdates, injectHeroImages, async (req, res) => {
+    if (res.locals.useNewBrand) {
+        res.render(path.resolve(__dirname, './views/home-rebrand'), {
+            heroImage: {
+                small: '/assets/images/hero/superhero-small.jpg',
```

Comment:
This image is a placeholder for now so I've not don't much to optimise the assets yet.